### PR TITLE
fpgad-vc: graceful shutdown models super-rsu flow

### PR DESCRIPTION
**Pull request by lzhan55**
Update the method used to remove the device driver to that of super-rsu:
* Write 1 to the device's reset node.
* Write 1 to the remove node of the root bridge's child. (removes
all VC devices)

When rescanning,
* Write "on" to the power/control node of the root bridge's child.
* Write 1 to the rescan node of the root bridge's child.

### CODE REVIEW

File: tools/base/fpgad/plugins/fpgad-vc/fpgad-vc.c [hunk 1]
```diff
@@ -500,40 +505,145 @@ STATIC fpga_result vc_unload_driver(vc_device *vc)
...
-		     "%d/remove", (int)fn);
+	// /sys/devices/pci0000:ae/0000:ae:00.0/0000:af:00.0/
+	// 0000:b0:09.0/0000:b2:00.0/reset
+	snprintf_s_s(path, PATH_MAX,
+		      "/sys/%s/reset", p);
 
-	LOG("writing 1 to %s to remove driver.\n", path);
-	if (file_write_string(path, "1\n", 2)) {
+	LOG("writing 1 to \"%s\" to reset the device.\n", path);
+	res = file_write_string(path, "1\n", 2);
```
Comment (tswhison) [inline tools/base/fpgad/plugins/fpgad-vc/fpgad-vc.c]: Please pick up the latest from master.  This code is re-introducing a bug.

Comment (lzhan55) [reply tools/base/fpgad/plugins/fpgad-vc/fpgad-vc.c]: I just did a cherry pick from master. Is it not correct?

Comment (lzhan55) [reply tools/base/fpgad/plugins/fpgad-vc/fpgad-vc.c]: I see. You made a new check-in a few minutes ago.

**Comment by lzhan55** [comment]
Opened a new PR to resolve conflicts. Will close this one.

**Pull request closed without merging**